docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
```

Comment:
We have an experimental docker branch: `feature/docker`. Will merge that as soon as I'm satisfied with the final result